ubuntu-jammy-docker-dbg/Dockerfile

```diff
@@ -0,0 +1,21 @@
+#
+# Ubuntu Jammy + Docker
+#
+# Instructions for docker installation taken from:
```

Comment:
This sentence doesn't apply to this file.

Reply:
This sentence is the same for previous versions: ubuntu-bionic-docker-dbg and ubuntu-focal-docker-dbg.
What would you prefer? Change all or keep this as the others.
What message would you like to have here?

Reply:
Oh, I see. Remove it from here please, we'll fix `ubuntu-focal-docker-dbg`, where it shouldn't be, later on.

Reply:
Done